test/e2e/tests/smart-transactions/mock-requests-for-swap-test.ts

```diff
@@ -288,18 +290,14 @@ const GET_TRANSACTION_BY_HASH_RESPONSE = {
 };
 
 export const mockSwapRequests = async (mockServer: MockttpServer) => {
-  await mockEthDaiTrade(mockServer);
+  await mockMultiNetworkBalancePolling(mockServer);
 
-  await mockServer
-    .forJsonRpcRequest({
-      method: 'eth_getBalance',
-      params: ['0x5cfe73b6021e818b776b421b1c4db2474086a7e1'],
-    })
-    .thenJson(200, {
-      id: 3806592044086814,
-      jsonrpc: '2.0',
-      result: '0x1bc16d674ec80000', // 2 ETH
-    });
```

Comment:
not needed, as we are mocking the balances using the mockMultiNetworkBalancePolling